### DISCUSSION TITLE
Benzodiazepines + 4mmc/Mephedrone

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#102] [Questions]
As expected, no issues with this combo. Found 2 papers talking about using benzos to manage agitation in people using mephedrone in emergency rooms.

Wood, D. M., Davies, S., Greene, S. L., Button, J., Holt, D. W., Ramsey, J., & Dargan, P. I. (2010). Case series of individuals with analytically confirmed acute mephedrone toxicity. Clinical Toxicology, 48(9), 924–927. https://doi.org/10.3109/15563650.2010.531021

Wood, D. M., Greene, S. L., & Dargan, P. I. (2011). Clinical pattern of toxicity associated with the novel synthetic cathinone mephedrone. Emergency Medicine Journal, 28(4), 280–282. https://doi.org/10.1136/emj.2010.092288

**Comment by utaninja** [comment]
I would presume that this would also be a factor-
Both can dull each other's effects, so if one wears off before the other it's possible to overdose due to the lack of counteraction.

How does this look?

Benzodiazepines + 4mmc =  Low Risk & Decrease
Both can dull each other's effects, so if one wears off before the other it's possible to overdose due to the lack of counteraction.

Wood, D. M., Davies, S., Greene, S. L., Button, J., Holt, D. W., Ramsey, J., & Dargan, P. I. (2010). Case series of individuals with analytically confirmed acute mephedrone toxicity. Clinical Toxicology, 48(9), 924–927. https://doi.org/10.3109/15563650.2010.531021
Wood, D. M., Greene, S. L., & Dargan, P. I. (2011). Clinical pattern of toxicity associated with the novel synthetic cathinone mephedrone. Emergency Medicine Journal, 28(4), 280–282. https://doi.org/10.1136/emj.2010.092288

**Reply by utaninja** [reply]
Would we want to mark this as caution instead? Seems to be the general way we do it for stimulants + depressants.

**Reply by utaninja** [reply]
So something like

Benzodiazepines + 4mmc = Caution
Both can dull each other's effects, so if one wears off before the other it's possible to overdose due to the lack of counteraction. Both substances affect the heart and may lead to further dangerous heart complications at higher dosages. Medical dosages are unlikely to be of concern for the general population.

Wood, D. M., Davies, S., Greene, S. L., Button, J., Holt, D. W., Ramsey, J., & Dargan, P. I. (2010). Case series of individuals with analytically confirmed acute mephedrone toxicity. Clinical Toxicology, 48(9), 924–927. https://doi.org/10.3109/15563650.2010.531021
Wood, D. M., Greene, S. L., & Dargan, P. I. (2011). Clinical pattern of toxicity associated with the novel synthetic cathinone mephedrone. Emergency Medicine Journal, 28(4), 280–282. https://doi.org/10.1136/emj.2010.092288

https://www.ncbi.nlm.nih.gov/books/NBK470159/

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/issues/136#issuecomment-3557866640
